fleetspeak/src/server/servertests/comms_test.go

```diff
@@ -257,3 +261,70 @@ func TestBlacklist(t *testing.T) {
 		t.Errorf("GetMessage([%v]) did not return expected RekeyRequest, want: %+v got: %+v", mid, msg, msgs[0])
 	}
 }
+
+// errorService is a Fleetspeak service.Service that returns a specified
+// error every time Service.ProcessMessage() is called.
+type errorService struct {
+	e error
```

Comment:
I know Go style guide encourages short names for variables, but I don't think it also holds for structure fields. I think `error` or at least `err` would be better. I get that this is only a test file, but readability should be maintaned everywere.

Reply:
Renamed.